include/Model.h

```diff
@@ -697,6 +697,14 @@ namespace ChimeraTK::Model {
 
   struct ReturnFirstHit : SearchOption {};
 
+  struct VisitOrder : SearchOption {
```

Comment:
It would be good to have a dedicated test for this new feature. Please write a ticket that we need to write one if not yet done...

Reply:
Will write a ticket for this